packages/nuxt/src/module.ts

```diff
@@ -79,12 +83,12 @@ export default defineNuxtModule<ModuleOptions>({
           consoleSandbox(() => {
             // eslint-disable-next-line no-console
             console.log(
-              '[Sentry] Your application is running in development mode. Note: @sentry/nuxt is in beta and may not work as expected on the server-side (Nitro). Errors are reported, but tracing does not work.',
+              '[Sentry] Your application is running in development mode. Note: @sentry/nuxt does not work as expected on the server-side (Nitro). Errors are reported, but tracing does not work.',
             );
           });
         }
 
-        if (moduleOptions.dynamicImportForServerEntry === false) {
+        if (moduleOptions.autoInjectServerSentry !== 'experimental_dynamic-import') {
```

Comment:
q: This we also want if the option is `undefined`, so that we can transpile the server config file, right?

Reply:
Yes, and this is the current behavior :)